packages/core/src/browser/window/browser-window-module.ts

```diff
@@ -15,8 +15,11 @@
  ********************************************************************************/
 
 import { ContainerModule } from 'inversify';
-import { DefaultWindowService, WindowService } from '../../browser/window/window-service';
+import { WindowService } from '../../browser/window/window-service';
+import { DefaultWindowService } from '../../browser/window/default-window-service';
+import { FrontendApplicationContribution } from '../frontend-application';
 
 export default new ContainerModule(bind => {
     bind(WindowService).to(DefaultWindowService).inSingletonScope();
+    bind(FrontendApplicationContribution).toService(WindowService);
```

Comment:
Q: `WindowService` is not necessarily a `FrontendApplicationContribution`, so shouldn't this be a service binding to `DefaultWindowService` (with the implication of an additional `DefaultWindowService` binding)